Userland/Applications/HexEditor/ValueInspectorModel.h

```diff
@@ -131,5 +132,6 @@ class ValueInspectorModel final : public GUI::Model {
     }
 
 private:
+    bool m_is_little_endian;
```

Comment:
Please default-initialise all trivial members (regardless of whether they're initialised in the ctor).
```suggestion
    bool m_is_little_endian = false;
```